pkgs/oauth2/lib/src/authorization_code_grant.dart

```diff
@@ -369,3 +364,19 @@ class _State {
   @override
   String toString() => _name;
 }
+
+enum OAuthPrompt {
```

Comment:
If this enumeration is from an oauth / oidc specification we need a reference to the relevant section.

Reply:
Needs documentation comment